collect_app/src/main/AndroidManifest.xml

```diff
@@ -280,5 +279,19 @@ the specific language governing permissions and limitations under the License.
             </intent-filter>
         </activity>
 
+        <activity
+            android:name=".external.FormUriActivity"
+            android:noHistory="true"
+            tools:ignore="AppLinkUrlError">
+
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <action android:name="android.intent.action.EDIT" />
```

Comment:
I'm not sure why we support `EDIT` for forms. It looks like we use `EDIT` as the action to open a form for entry from the "Fill Blank Form" page, but I don't think there is really any need for that - it could launch the Activity directly. Anyway, that's a battle for another day.

Reply:
I believe that the default when launching the activity directly is indeed edit. I think this is mostly doe symmetry for external clients.

Reply:
Ah nice, I had incorrectly assumed there wasn't docs for this. Yeah, I think ultimately I don't mind whether we use `EDIT` or `VIEW` to mean "fill a blank form" but it doesn't make sense to support both. Given the docs, I think I should change the tests to focus on the `EDIT` action rather than `VIEW`. I could also change shortcuts to use `EDIT`. What do you think?